step2-03/README.md

```diff
@@ -1,31 +1,48 @@
-# Step 2.3: Theming and Styling
+# Step 2.3: Theming and styling with UI Fabric
 
 [Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
 
-Theming and Styling with UI Fabric. In this section, we will illustrate how to utilize some of the built-in theming and styling features right inside UI Fabric component library. 
+In this section, we will illustrate how to use some of the built-in theming and styling features of the UI Fabric component library.
 
-For advanced or non-Fabric component scenarios, UI Fabric also exposes its own CSS-in-JS library called `mergeStyles` that is very performant compared with other similar libraries. A CodePen that illustrates what `mergeStyles` does: https://codepen.io/dzearing/pen/jGdgrE?editors=1011
+These are the theming and styling methods that we will focus on in this step:
 
-These are the areas that we will focus on in this step:
-
-1. Theming with Fabric using `<Customizer>` component
+1. Theming using the `<Customizer>` component
 2. Customizing themes and loading with `loadTheme()`
-3. Customizing Fabric Components `styles` prop
-4. CSS-in-JS with mergeStyles
+3. Customizing Fabric components via the `styles` prop
+4. CSS-in-JS with `mergeStyles`
+
+The first three methods only work with Fabric components, but the fourth, `mergeStyles`, can be used in other projects as well.
+
+## 1. Applying Fabric themes using `<Customizer>`
+
+One way to apply a theme is by wrapping the components to be themed with a `<Customizer>` component. `Customizer` propagates the theme down to children through the [React Context](https://reactjs.org/docs/context.html) mechanism.
+
+There are some predefined themes within Fabric already, like Fluent (which will become the default in the next major release), MDL2, Azure, and some other sample themes like Teams.
+
+The following code (simplified from `demo/src/components/TodoApp.tsx`) shows an example of applying the Fluent theme to our todo app using `Customizer`.
+
+```tsx
+import { Customizer } from 'office-ui-fabric-react';
+import { FluentCustomizations } from '@uifabric/fluent-theme';
 
-## Fabric Theming and Styling
```

Comment:
Removed the Fabric/non-Fabric distinction mainly because it messes up the numbering (and noted inline with mergeStyles that it's more advanced and works outside fabric)